labs/b5.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Lab 5 - Networking 101
+layout: lab
+---
+
+## Overview
+It is undeniable that the internet is an important system that has redefined our world. The ability to develop networks and allow devices to communicate is critical to value of modern day computer systems.  This lab will take a look into the basics of computer networking and then examine networks through the perspective of a sysadmin.
+
+We will be using web browsing as an analogy to understand the basics of networking. What exactly happens when I go web browsing for cat pictures?
+
+But first let's take a short dive into the details of networking.
+
+---
+
+## OSI Model
+To lay the groundwork for understanding networking, we'll first turn to the Open Systems Interconnection model (OSI).  The OSI model describes modern computer systems by partitioning them into several 'layers':
+1. **Physical Layer**
+This layer deals with the physical transmission of the data such as passing electrical signals over a fiber optic cable or radio frequencies for wireless.
```

Comment:
You may need to indent these out one more to get them to display properly

Reply:
Seems fine on my end via github and hackmd

Reply:
Seems fine to me, it just isn't indented under the numbered list but is instead inline after the bolded text like so:

<img width="979" alt="screen shot 2018-02-27 at 5 32 40 pm" src="https://user-images.githubusercontent.com/1523594/36765051-3da02f0e-1be4-11e8-980e-e0164d519b4f.png">